tests/e2e/Services/VCS/VCSConsoleClientTest.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace Tests\E2E\Services\VCS;
+
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\SideConsole;
+use Utopia\App;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Role;
+use Utopia\VCS\Adapter\Git\GitHub;
+use Utopia\Cache\Adapter\None;
+use Utopia\Cache\Cache;
+
+class VCSConsoleClientTest extends Scope
+{
+    use ProjectCustom;
+    use SideConsole;
+
+    public ?string $providerRepositoryId = null;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->providerRepositoryId = '705764267';
+    }
```

Comment:
```suggestion
    public string $providerRepositoryId = '705764267';
```

We might not need setup for this.